code/modules/client/preference_setup/vore/07_traits.dm

```diff
@@ -92,7 +93,8 @@
 	. += "<b>Custom Species</b> "
 	. += "<a href='?src=\ref[src];custom_species=1'>[pref.custom_species ? pref.custom_species : "-Input Name-"]</a><br>"
 
-	if(pref.species == SPECIES_CUSTOM || pref.species == SPECIES_XENOCHIMERA)
+	var/datum/species/selected_species = all_species[pref.species]
+	if(selected_species.selects_bodytype)
```

Comment:
Good change, but you need to rewrite this statement here otherwise they will get sanitized:
https://github.com/VOREStation/VOREStation/pull/3885/files#diff-5cc81ee95e08298782b9f64fe5685af5R76

Reply:
Done. Nice catch.